### PR TITLE
Archives, uc center plugin, and single posts

### PR DESCRIPTION
**Pull request by ghost**
#reviewmerge @jeremyfelt @philcable Create templates and styles for archives, uc center people and organizations, and single posts. 

Would one of you two help me get the category title to show up above the uc center people and organization archive pages?

### CODE REVIEW

File: articles/post-wsuwp_uc_entity.php
Comment (philcable) [inline articles/post-wsuwp_uc_entity.php:6]: This `<?php else: ?>` isn't necessary if you don't need to output anything for archive views. And if that is indeed the case, you might consider moving the `is_single()` check outside of the `<header>` element (example below) so we're not outputting empty markup.

```
<?php if ( is_single() ) : ?>
<header class="article-header">
...
</header>
<?php endif; ?>
```

File: articles/post-wsuwp_uc_entity.php
Comment (philcable) [inline articles/post-wsuwp_uc_entity.php:28]: I think we're missing the closing tag for the `.flex-body` div here.

File: articles/post-wsuwp_uc_person.php
Comment (philcable) [inline articles/post-wsuwp_uc_person.php:43]: Not a big deal, but adjusting the indents here would make it easier to tell if all the HTML tags are properly nested and closed.

File: parts/single-layout-wsuwp_uc_person.php
Comment (philcable) [inline parts/single-layout-wsuwp_uc_person.php:5]: Consider adjusting the indents for readability in this file, too.

**Review by philcable — COMMENTED** [review]
Looking good! I left a couple comments about some very minor things. I'll let @jeremyfelt take a quick look through it too before merging just in case I missed anything.

**Review by jeremyfelt — APPROVED** [review]

**Pull request merged**
